cypress/integration/navigationFlows/homePageLeftSidebar.spec.js

```diff
@@ -0,0 +1,31 @@
+describe('Home Page Left Sidebar', () => {
```

Comment:
test (blocking): The test runs fine, but I am unable to rerun it. It fails on subsequent runs waiting for the homepage. I'm not sure why yet. This is fine for CI, but for someone writing tests, this will constantly fail after the first time it's run.

![image](https://user-images.githubusercontent.com/833231/109224854-837afe00-778a-11eb-86f3-68537b71017e.png)

Reply:
Hmm not sure what happened but I can't reproduce it, at least after I merged master and fixed conflicts. Maybe that was related somehow?